Cargo.toml

```
@@ -18,6 +18,7 @@ base64 = [ "base64-compat" ]
 unstable = []
 rand = ["secp256k1/rand-std"]
 use-serde = ["serde", "bitcoin_hashes/serde", "secp256k1/serde"]
+serde-std = ["serde/std"]
```

Perhaps we should turn on std in bitcoin_hashes as well?

```suggestion
serde-std = ["serde/std", "bitcoin_hashes/serde-std"]
```

---

That'd force-include `bitcoin_hashes`, so no. If we require it anywhere we may need `serde-bitcoin-hashes-std` :(

---

We could go down a rabbit hole, exhaustively reexporting the `foo/std` sub features. Its not clear to me where to draw the line, so I kept this PR to the minimal change to fix broken builds for `no-std` users of Serde.

I'm open to suggestions if you'd prefer I apply this same logic across some/all other features too

---

Sorry for the dud suggestion, seems my brain cannot easily model this matrix of features across crates.